docs/src/contributors/index.rst

```diff
@@ -0,0 +1,89 @@
+Contributors Guide
+==================
+
+Thank you for your interest in contributing to **senfd**! Please follow the
+guidelines below to ensure a smooth and effective collaboration.
+
+Commit Message Guidelines
+-------------------------
+
+**senfd** follow the
+`Conventional Commits v1.0.0 <https://www.conventionalcommits.org/en/v1.0.0/>`_ 
+specification for commit messages. A clear and consistent commit message format
+helps us understand the history of **senfd**. Each commit message should:
+
+- Start with a type (e.g., feat, fix, refactor) and a brief summary.
+- Be written in the imperative mood.
```

Comment:
Is it really mood? I am not sure myself.

Reply:
yup :)